ts-client/src/amm/index.ts

```diff
@@ -170,6 +171,51 @@ export default class AmmImpl implements AmmImplementation {
     };
   }
 
+  public static async createConfig(
```

Comment:
create config is permissionless as well? I thought only created by admin

Reply:
it only created by admin

Reply:
Needed it in local test to create config.